analysis/topeft_run2/run_analysis.py

```diff
@@ -223,6 +223,7 @@ def LoadJsonToSampleName(jsonFile, prefix):
 
     # Extract the list of all WCs, as long as we haven't already specified one.
     if len(wc_lst) == 0:
+        print("WARNING: specify an ordered list of wcs for scalings, otherwise the order would be arbitrary.")
```

Comment:
Same with this line

Reply:
It loads from `histEFT` now so I will remove this line. Thanks for catching it.